src/Components/Events/Cards/index.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import {Text, StyleSheet, View, Linking} from 'react-native';
+import ScaledImage from '../../ScaledImage';
+import { withCard } from './../../../Decorators/Card';
+import Badge from './CardBadge';
+
+const EventCard = ({ props, isOver }) => {
```

Comment:
`isOver` is not being used anywhere. Please remove it if not required

Reply:
isOver is required